arch/univariate/distribution.py

```diff
@@ -301,6 +352,26 @@ def ppf(self, pits, parameters=None):
         pits = asarray(pits)
         return stats.norm.ppf(pits)
 
+    def moment(self, h, parameters=None):
+        return stats.norm.moment(h)
+
+    def partial_moment(self, h, z=0, parameters=None):
+        """
```

Comment:
This overwrites previous docstring, so needs full docsting.  Can also use composition to "build" the string and then poke it on.